src/patternfly/layouts/Flex/flex.scss

```diff
@@ -167,41 +155,68 @@ $pf-flex-spacer-map: (
   @mixin flex-modifiers($mod: null) {
     // display
     &.pf-m-flex#{$mod} {
-      display: flex;
+      display: var(--pf-l-flex--Display);
     }
 
     &.pf-m-inline-flex#{$mod} {
-      display: inline-flex;
+      --pf-l-flex--Display: inline-flex;
     }
 
     // flex-direction
     &.pf-m-column#{$mod} {
       flex-direction: column;
+      align-items: normal;
+
+      > * {
+        margin: 0 0 var(--pf-l-flex--spacer) 0;
+      }
     }
 
+    // flex-direction
     &.pf-m-column-reverse#{$mod} {
       flex-direction: column-reverse;
+      align-items: normal;
+
+      > * {
+        margin: var(--pf-l-flex--spacer) 0 0 0;
+      }
     }
 
     &.pf-m-row#{$mod} {
       flex-direction: row;
+      align-items: var(--pf-l-flex--m-row--AlignItems);
+
+      // unset styling set for .pf-m-column
+      > * {
+        margin: 0 var(--pf-l-flex--spacer) 0 0;
+      }
     }
 
     &.pf-m-row-reverse#{$mod} {
       flex-direction: row-reverse;
+      align-items: var(--pf-l-flex--m-row-reverse--AlignItems);
+
+      // unset styling set for .pf-m-column
+      > * {
+        margin: 0 0 0 var(--pf-l-flex--spacer);
+      }
     }
 
     // flex-wrap
     &.pf-m-wrap#{$mod} {
       flex-wrap: wrap;
     }
 
+    &.pf-m-wrap-reverse#{$mod} {
+      flex-wrap: wrap-reverse;
+    }
+
     &.pf-m-nowrap#{$mod} {
       flex-wrap: nowrap;
     }
 
-    &.pf-m-wrap-reverse#{$mod} {
-      flex-wrap: wrap-reverse;
+    &.pf-m-flex-none#{$mod} {
```

Comment:
Can you add this to the documentation? You might also consider moving it by the other shorthand `flex` properties, like after `.pf-m-flex-default`